### PR TITLE
fix(navbar):  navbar will display if no guide exist

### PR DESCRIPTION
**Pull request by FredericEspiau**
### Description of the Change

The menu crashed if the guides did not exist in the constants file.

### CODE REVIEW

File: client/app/components/manager-navbar/manager-navbar.module.js [hunk 1]
```diff
@@ -11,79 +11,86 @@ angular.module("App").run((LANGUAGES, managerNavbar, User, constants, translator
...
+                } else {
+                    OtrsPopupService.toggle();
+                }
+            }
+        }, {
+            label: translator.tr("common_menu_support_list_ticket"),
+            url: "#/support"
+        }];
+
+        if (currentSubsidiaryURLs != null) {
```
Comment (AxelPeter) [inline client/app/components/manager-navbar/manager-navbar.module.js]: !== ?

Comment (Apollidore) [reply client/app/components/manager-navbar/manager-navbar.module.js]: `x != null` === `x != null && x != undefined`

Comment (jleveugle) [reply client/app/components/manager-navbar/manager-navbar.module.js]: I understand @AxelPeter feedback.

What do you think about this syntax : `if(!_.isEmpty(currentSubsidiaryURLs)) ` ? :)

Comment (FredericEspiau) [reply client/app/components/manager-navbar/manager-navbar.module.js]: I actually should have adopted this syntax to start with.

Comment (jleveugle) [reply client/app/components/manager-navbar/manager-navbar.module.js]: Can you fix these PR plz ? :)

**Pull request closed without merging**